Samples.Swift/Samples.Swift/ViewControllers/CheckoutViewController/CheckoutViewController.swift

```diff
@@ -0,0 +1,444 @@
+//
+//  CheckoutViewController.swift
+//  paypalcheckout-iOS-swift-sample
+//
+//  Created by Jax DesMarais-Leder on 3/10/21.
+// Copyright © 2021 PayPal. All rights reserved.
+//
+
+import UIKit
+import PayPalCheckout
+
+class CheckoutViewController: UIViewController, AddItemViewControllerDelegate {
+
+  let tableView = UITableView()
+  let checkoutButton = UIButton()
+  var checkoutFlowOption = UISegmentedControl()
+  var items = [PurchaseUnit.Item]()
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    setupSampleItem()
+    setupUI()
+    setupTableView()
+    setupConstraints()
+  }
+
+  @objc
+  func tapCheckout() {
+    if items.count == 0 {
+      let ac = UIAlertController(title: "Cannot checkout", message: "Please add at least 1 item to the cart", preferredStyle: .alert)
+      ac.addAction(UIAlertAction(title: "Ok", style: .default))
+      self.present(ac, animated: true)
+    }
+
+    Checkout.setOnApproveCallback { approval in
+      self.processOrderActions(with: approval)
+    }
+
+    Checkout.setOnCancelCallback {
+      print("Checkout cancelled")
+    }
+
+    Checkout.setOnErrorCallback { errorInfo in
+      print("Checkout failed with error info \(errorInfo.error.localizedDescription)")
+    }
```

Comment:
If we are starting the SDK immediately following, i think it makes more sense to keep the closures in the `start` call instead of declaring them separately here

Reply:
I had done it in this way to show off the multiple ways callbacks could potentially be set but I can certainly move it all within start if we plan to outline this elsewhere. Thoughts?

Reply:
If we want to show off multiple ways of setting callbacks, we should do it in different flows. mixing configuration paths within the same flow i think is more confusing than helpful

Reply:
Sounds good, I can put this all in start in that case! We will also want to update this in the objc app @minhthenguyen

Reply:
Since both onCancel and onError don't change based on the case, does it make sense to move setting these callbacks to the top of the app lifecycle instead? We could also potentially do the same with onApprove, but onApprove relies on a function within this scope so would likely get confusing. Setting these on Checkout reduces a bit of code duplication as `Checkout.start` is slightly different for each case below within the `startNativeCheckout` function.

Reply:
I think normally merchants would wanna handle onCancel and onError in their ViewController (show dialog, or go back to their checkout cart screen or something) so maybe I think it's good to leave them inside the view controller :smile:

Reply:
to reduce the code duplication probably we can set these callbacks inside `viewDidLoad()` instead of on button tap. 
I think either that or having callbacks in start is nice! :smile:

Reply:
I'm fine with either! I don't know how common it will be to have multiple `Checkout.start`s so it may be fine to duplicate as we have 2 different flows where most folks would use one. I just want to make sure we are consistent across the 2 apps 😄

Reply:
Just circling back on this - i think we should still move to consolidate this into the `start` function, similar to the objective c sample app. 

It could be good to provide a follow up PR to show a payment button and configure the SDK that way as well, but for the sake of clarity let's collect them all

Reply:
Re: setting them at the top - i think that aligns with setting the closures in the Config object, which is also possible. my thought is for each demonstrated flow in the sample app should use a single 'path' for setting the callbacks, mostly for clarity. What do you think?

Reply:
Yep, looks like the objc app was updated after this, so can update here to align with the objc updates. Just wanted to make sure we were doing the same thing in both! I'll update the swift app to align with the updates that were made in objc!